sources/javascripts/concat/global/main.js

```diff
@@ -451,6 +464,10 @@
       handleProductCountChange(e, true);
     });
 
+    $(document).on('voog:shoppingcart:contentschanged', function (e) {
+      handleProductCountSync(e);
+    })
```

Comment:
Ja siis ei oleks siin samuti seda `e`-d vaja:
```suggestion
    $(document).on('voog:shoppingcart:contentschanged', function () {
      handleProductCountSync();
    })
```